packages/js/test/unit/server.test.ts

```diff
@@ -299,4 +299,29 @@ describe('server client', function () {
       expect(request.isDone()).toBe(true)
     })
   })
+
+  describe('__plugins', function () {
+    it('exported singleton includes plugins', function () {
+      Singleton.configure({ apiKey: 'foo' })
+      expect(Singleton.config.__plugins.length).toBe(2)
+    })
+    
+    it('clients produced via factory don\'t include plugins', function () {
```

Comment:
We are debating whether this is what we want in https://github.com/honeybadger-io/honeybadger-js/issues/1181, but I figured I'd document the current behavior.